common/rfb/ServerCore.cxx

```diff
@@ -101,3 +101,21 @@ rfb::BoolParameter rfb::Server::queryConnect
 ("QueryConnect",
  "Prompt the local user to accept or reject incoming connections.",
  false);
+rfb::IntParameter rfb::Server::dynamicQualityMin
```

Comment:
You can put these in `EncodeManager.cxx`. The ServerCore stuff is more things that are used in various places. These new settings are very specific to the encoder.